ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -369,6 +369,13 @@ object Config {
             "The maximum number of commands waiting to be submitted for each party. Overflowing this threshold will cause back-pressure, signaled by a RESOURCE_EXHAUSTED error code. Default is 512."
           )
 
+        opt[Boolean]("command-deduplication-enabled")
+          .optional()
+          .action((value, config) =>
+            config.copy(commandConfig = config.commandConfig.copy(deduplicationEnabled = value))
+          )
+          .text("Enable or disable command deduplication. Default is enabled.")
```

Comment:
I don't think it's a good idea to expose this as a CLI option.

We will switch this from enabled to disabled when we switch from v1 of the participant state API to v2.

Right now it's enough to verify this works in tests.